xl2times/transforms.py

```diff
@@ -2334,7 +2369,12 @@ def eval_and_update(
             # Overwrite (inplace) the column given by the attribute (translated by attr_prop)
             # with the value from row
             # E.g. if row['attribute'] == 'PRC_TSL' then we overwrite 'tslvl'
-            table.loc[rows_to_update, attr_prop[row["attribute"]]] = row["value"]
+            if row["attribute"] not in attr_prop:
+                logger.warning(
+                    f"Unknown attribute {row['attribute']}, skipping update."
+                )
+            else:
+                table.loc[rows_to_update, attr_prop[row["attribute"]]] = row["value"]
```

Comment:
Had to add this check to get past here with AusTimes.
Not sure if this should be picked up in an earlier validation step again though?

Reply:
Good point! I think we can move this to an earlier validation step later.

Reply:
Actually any chance you could share what are the unknown attributes that appear here in AusTIMES? In addition to an input error, it could happen that something is introduced during an earlier transform...

Reply:
`attr_prop` contains all of the attributes that are valid in this table type.

Reply:
The missing attr_prop here was called 'ACTCOST', 
If that doesn't mean anything to it's probably an AusTIMES specific thing.

Reply:
Well, actually `ACTCOST` is a valid Veda alias for the TIMES attribute `ACT_COST` and we have it in `veda-attr-defaults.json`. However, as far as I know, it should not appear in a `~TFM_INS-TXT` table. Unless something has changed, only the 6 attributes in `attr_prop` are valid in this table type.

Reply:
Ok, I just double checked and I'm getting a warning for `ACT_BND` now too.
It's very possible that this is human error, but it's also something that VEDA apparently handles.
What do you suggest here?

Reply:
I'd say it is fine for now. Later on we can add a check in an earlier transform supplemented by warnings.